user/pages/03.commerce2/01.getting-started/01.using-composer/docs.md

```diff
@@ -11,7 +11,7 @@ distributable libraries that can be shared and integrated by others. This is
 possible in part by the PHP Framework Interoperability Group (FIG) and [PSR-4]

```

Comment:
Whomp, my "SEO" hacks broke this page :( https://pr-236-4wo45ci-boljpk23hsit6.eu.platform.sh/commerce2/developer-guide/install-update/using-composer

I'll try to fix.

Reply:
This is fixed, but the page shows a 404 on Platform.sh but not locally, which is weird.